### DISCUSSION TITLE
The frame rendering frame number is very low. What is the reason?

### DISCUSSION THREAD

**Discussion by Lydanne** [fltk-rs/fltk-rs#1342] [Q&A]
Please watch the video:

https://user-images.githubusercontent.com/39021696/213654241-29689a1a-375c-4cfc-a11b-3b776d2eeec5.mov

**Comment by Lydanne** [comment]
The screen recording effect is not obvious.

**Reply by MoAlyousef** [reply]
Can't really say without looking at any code. Are you building in release mode?

**Reply by Lydanne** [reply]
git repo: https://github.com/drawpanel/drawpanel

Currently, it is only in the development mode.

**Reply by MoAlyousef** [reply]
Why is there a multiline input. Its handle method is causing the draw frame to be repeated redrawn. It causes high cpu usage.

**Reply by Lydanne** [reply]
Because I need to implement the function of text.
Is it because of multiline input?

**Reply by Lydanne** [reply]
It is not the reason for multiline input. I have remove all the relevant codes, It's still like this

**Reply by MoAlyousef** [reply]
Something is causing the draw frame to be redrawn repeatedly even when idle.
